app/services/google_scraping_service.rb

```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
+
+require 'httparty'
+require 'nokogiri'
+
+class GoogleScrapingService
+  attr_reader :keyword_id, :keyword
```

Comment:
do we need to set those attr as public? should we move to private

```
private

attr_reader :keyword_id, :keyword
```

Reply:
Fixed in bd7d3f14a56f68783ac2a0a85b1806e3c3cee546